.buildbot.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/sh
+
+set -eu
+
+# lint check
+cargo fmt --all -- --check
```

Comment:
rust and cargo are not in the default docker image.

I'd recommend copying that bit from another repo, e.g.:
https://github.com/softdevteam/lang_tester/blob/a78e4bc037ec300a1767dba587c3d44bea86e009/.buildbot.sh#L5-L11

Reply:
Also we should test release mode too.

Reply:
ah ok

Reply:
I moved rust and cargo installation to the docker file, cause I prefer to run tests locally directly via the sh script.
Would that work with bors integration?